src/pages/workspace/WorkspacePageWithSections.js

```diff
@@ -86,7 +86,8 @@ class WorkspacePageWithSections extends React.Component {
 
     fetchData() {
         const achState = lodashGet(this.props.reimbursementAccount, 'achData.state', '');
-        BankAccounts.fetchFreePlanVerifiedBankAccount('', achState);
+        // BankAccounts.fetchFreePlanVerifiedBankAccount('', achState);
+        BankAccounts.openWorkspaceView();
```

Comment:
Don't worry about this. I am hoping to remove this whole component in this PR soon